### PR TITLE
To Do List structure

### PR DESCRIPTION
**Pull request by GOlivierNation**
**What Is In This PR:**
*************************
1. Set up a new project with webpack that is based on the webpack exercise.
2. Create an empty To Do List placeholder
3. Create an index.js file and set an array of some simple to do tasks (array of objects).
4. Write a function to iterate over the tasks array and populate an HTML list item element for each task.
5. On page load render the dynamically created list of tasks in the dedicated placeholder.
6. Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader.
***************************************************************************************************************************

### CODE REVIEW

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @GOlivierNation, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments below under the review._

- [ ] Please you project is missing the `package.json` file, which is required as it contains the list of `devDependencies` for your project. It is required for setting up `webpack` and all the scripts of running your app. Please kindly add the file with  various `devDependencies`  and scripts for setting up and serving you app. This [article](https://webpack.js.org/guides/getting-started/) should be resourceful. 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GOlivierNation** [comment]
> Hi @GOlivierNation,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> ## Required Changes recycle
> 
> _Check the comments below under the review._
> 
>     * [ ]  Please you project is missing the `package.json` file, which is required as it contains the list of `devDependencies` for your project. It is required for setting up `webpack` and all the scripts of running your app. Please kindly add the file with  various `devDependencies`  and scripts for setting up and serving you app. This [article](https://webpack.js.org/guides/getting-started/) should be resourceful.
> 
> 
> ## Optional suggestions
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Sorry @t-yanick you can check now the requested changes are added

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+];
+
+const todoList = document.getElementById('todo-list');
+todo.forEach((todo) => {
+  todoList.innerHTML += `
+      <li><hr></li>
+      <li class="todo">
+          <div>
+              <input type="checkbox" name="checkbox-${todo.index}" ${todo.completed ? 'checked' : 'unchecked'}>
+              <h2>${todo.description}</h2>
```
Comment (omar-muhamad) [inline src/index.js:28]: - [ ] [Optional] It would be better semantically if you replace this `<h2>` tag with `<p>` tag.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Approved ✔
### Hi @GOlivierNation,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Please consider doing the first `optional` suggestion, I marked it `optional` because the previous reviewer forgot to mention it.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- [ ] [Highly Recommended] Please add `.gitignore` file and add `/node_modules` directory to it to prevent uploading unwanted files to your repo.

- Please check the inline comments below ⬇.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request closed without merging**